pyproject.toml

```diff
@@ -52,11 +52,12 @@ ruff = {version="==0.2.1", optional=true}
 sphinx = {version="*", optional=true}
 sphinxcontrib-openapi = {version="^0.8.3", optional=true}
 mypy = {version="^1.10.1", optional=true}
+matplotlib = {version="^3.7.1", optional=true}
```

Comment:
```suggestion
```

Reply:
It is just so helpful to be able to plot and to have the plot in a place where I know all of the tests are.